src/py-opentimelineio/opentimelineio/console/otiotool.py

```diff
@@ -133,6 +136,19 @@ def main():
 
     # Final Phase: Output
 
+    if args.list_versions:
+        print("Available versions for --downgrade FAMILY:VERSION")
+        for family, mapping in otio.versioning.full_map().items():
+            for label in mapping.keys():
+                print(f"  {family}:{label}")
+
+    if args.downgrade:
+        if ":" in args.downgrade:
+            label = args.downgrade
+        else:
+            label = "OTIO_CORE:" + args.downgrade
```

Comment:
Should a log be printed to help the user know which version was picked up?

Reply:
Nah, I think it is clear enough as-is.